### DISCUSSION TITLE
Changing currency to GBP makes products disappear

### DISCUSSION THREAD

**Discussion by weefunker** [solidusio/solidus#4730] [Troubleshooting] [answered]
Fresh install added some products on production. 

https://files.catbox.moe/k7jm0g.mp4

**Comment by jarednorman** [comment, marked as answer]
I've moved this to the troubleshooting board. If we determine there's a bug with Solidus here, we can create an issue with the specific details.

Can you provide more information:
- What version of Solidus?
- What have tried?
- What have you learned debugging this?
- Does this happen with other currencies?
- How are the products configured?
- Do the products have GBP prices?

**Reply by weefunker** [reply]
What version of Solidus? latest
What have tried? changing currency to GBP on each field on the products
What have you learned debugging this? Solidus? A lot more than I expected 
Does this happen with other currencies? Yes every other currency 
How are the products configured? default config, some with variants, some without 
Do the products have GBP prices? yes but the internal master price wont change USD

**Reply by jarednorman** [reply]
> Do the products have GBP prices? yes but the internal master price wont change USD

Does creating a master price in GBP have any effect?

**Reply by weefunker** [reply]
How do you go about that Jared? Thanks for your help

**Reply by weefunker** [reply]
There doesn't seem to be an option to set the master price to anything other than USD. Also, I just noticed this happens locally as well.

**Reply by weefunker** [reply]
https://github.com/spree/spree/issues/3473 seems to have happened someone else some while back

**Reply by jarednorman** [reply]
Go into the prices tab, hit new price, select the master variant, select GBP, enter a value, hit create.

**Reply by weefunker** [reply]
Trying now

**Reply by weefunker** [reply]
The options to change the price from anything other than the default (USD) are greyed out/not selectable

**Reply by weefunker** [reply]
https://files.catbox.moe/a6axvm.mp4

**Reply by jarednorman** [reply]
My instructions were to add a new master price, not change the existing one. You cannot change the currency of an existing price.

**Reply by weefunker** [reply]
Got it, might want to include in docs 

# config/initializers/spree.rb

Spree.config do |config|
  config.currency = 'GBP'
end
